HMD/Assets/Scenes/scripts/TSScConnection.cs

```diff
@@ -44,6 +44,18 @@ public void ConnectToHost(string host, int team_number)
         // Test Connection
         StartCoroutine(GetRequest(this.url));
     }
+    // Connect with port
+    public void ConnectToHost(string host, int team_number, int port)
+    {
+        this.host = host;
+        this.port = port.ToString();
+        this.team_number = team_number;
+        this.url = "http://" + this.host + ":" + this.port;
+        Debug.Log(this.url);
+
+        // Test Connection
+        StartCoroutine(GetRequest(this.url));
```

Comment:
looks good, will we be able to use this for the LMCC server i wrote as well? It also broadcasts over Local Networks and follows the JSON data formatting, so something to interface with that would be a nice inclusion (i'll add this to the tasks board as well)

Reply:
we need this capability to transmit data more efficiently between the two components of this project